src/main/java/com/google/sps/data/Task.java

```diff
@@ -0,0 +1,16 @@
+package com.google.sps.data;
+
+public final class Task {
```

Comment:
instead of "Task", can you give this class a name that reflects the data it holds?